deps/verifier/src/lib.rs

```diff
@@ -181,7 +181,7 @@ pub trait Verifier {
 }
 
 /// Padding or truncate the given data slice to the given `len` bytes.
-fn regularize_data(data: &[u8], len: usize, data_name: &str, arch: &str) -> Vec<u8> {
+pub fn regularize_data(data: &[u8], len: usize, data_name: &str, arch: &str) -> Vec<u8> {
```

Comment:
is this related to this PR?

Reply:
Yes as I delete the `all-attester` from default feature, by default nothing will import this code so the lint would warn. By making it `pub` the lint can be ok